app/models/tv_actor_credit.rb

```diff
@@ -1,35 +1,32 @@
 class TVActorCredit
-  def initialize(show_name, show_id, actor_name, actor_id, character, seasons, episodes)
+  attr_accessor :show_name, :show_id, :actor_name, :actor_id, :character, :seasons, :episodes, :profile_path, :known_for
```

Comment:
I've added `profile_path` and `known_for` as I'm planning to incorporate them in a subsequent PR for an `episode` show page.